### PR TITLE
fix: drop unsafe iOS keychain accessibility opts

### PR DESCRIPTION
**Pull request by demetris-manikas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,9 +108,10 @@ On iOS it is possible to configure the accessibility of the keychain by setting
  * AfterFirstUnlockThisDeviceOnly
  * WhenUnlocked
  * WhenUnlockedThisDeviceOnly
+ * WhenPasscodeSetThisDeviceOnly (this option is available only on iOS8 and later)
+but NOT to one of the following:
  * Always
  * AlwaysThisDeviceOnly
```
Comment (dcousens) [inline README.md]: Why not disable these then?

Comment (demetris-manikas) [inline README.md]: @dcousens They are disabled by the changes made in  SecureStorage.m.

Comment (dcousens) [inline README.md]: I don't think you should list them then.
If you want to warn others who might attempt to use them,  put it in a following paragraph.
Not in the list of options.

It reads very confusingly in the rich text.

Comment (demetris-manikas) [inline README.md]: Maybe. @ggozad Will take the decision. I made this change just to make him correct it...

Comment (ggozad) [inline README.md]: Yes please change the PR and remove the insecure options all together, no point in having them there.

**Comment by ggozad** [comment]
Please remove the unsafe options and we are good to merge.

**Comment by demetris-manikas** [comment]
README updated. 
I have not tested any of this. 
Maybe an error should be thrown when the configuration option is not present in detail allowed values. That would make the code testable and would inform developers that the setting they is not used

**Comment by demetris-manikas** [comment]
README updated. 
I have not tested any of this. 
Maybe an error should be thrown when the configuration option is not present in detail allowed values. That would make the code testable and would inform developers that the setting they is not used

**Comment by ggozad** [comment]
I will test this next week. I doubt anyone would be using the options removed, no need for checks.

**Comment by demetris-manikas** [comment]
the check is already there. The question is whether to throw an error or silently use the default when the config option is wrong. I believe the former is better.
And another matter that came to mind. 
Should the README be updated to state which is the default value used?

**Comment by dcousens** [comment]
Throw.

**Comment by ggozad** [comment]
@demetris-manikas yes please throw and update the README.

**Pull request merged**